code/WorkInProgress/Mini/ATM.dm

```diff
@@ -362,6 +363,52 @@ log transactions
 					playsound(loc, 'sound/items/polaroid1.ogg', 50, 1)
 				else
 					playsound(loc, 'sound/items/polaroid2.ogg', 50, 1)
+			if("vending_machine")
+				if(authenticated_account)
+					if(world.timeofday < lastprint + PRINT_DELAY)
+						usr << "<span class='notice'>The [src.name] flashes an error on its display.</span>"
```

Comment:
You don't need to change usr.
All the code in this section uses usr, and it's a Topic, so it's pretty much standard.